test/Sdk.Generator.Tests/Sdk.Generator.Tests.csproj

```diff
@@ -23,6 +26,21 @@
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+    <PackageReference Include="Microsoft.NETCore.Platforms" Version="7.0.4" />
```

Comment:
Can you try without the `Microsoft.NETCore.*` dependencies?

Reply:
That was one of the failing one. see [line 93 here](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=161094&view=logs&j=1372d9e0-5cd3-5ef5-5e82-7ce7a218d320&t=2f6a0721-e0ae-596e-eb00-8b34942a5a7f)